js/src/dashboard/all-programs-table-card/edit-program-button/edit-program-prompt-modal/index.scss

```diff
@@ -1,5 +1,5 @@
 .gla-edit-program-prompt-modal {
 	@include break-small {
-		max-width: 360px;
+		max-width: $modal-min-width;
```

Comment:
❔ Using modal-**min**-width variable for the **max**-with property looks a bit confusing.